Amazon.Ion.ObjectMapper/IonObjectSerializer.cs

```diff
@@ -25,6 +25,19 @@ public IonObjectSerializer(IonSerializer ionSerializer, IonSerializationOptions
 
         public override object Deserialize(IIonReader reader)
         {
+            var ionConstructors = targetType.GetConstructors(BINDINGS).Where(IsIonConstructor);
+            if (ionConstructors.Any())
+            {
+                if (ionConstructors.Count() > 1)
+                {
+                    throw new NotSupportedException(
+                        $"More than one constructor in class {targetType.Name} " +
```

Comment:
Can you reword this to be a bit more clear, something like "Only one constructor in class <blah> may be annotated with the ..."

Reply:
Done.